packages/table/src/common/errors.ts

```diff
@@ -5,16 +5,7 @@
  */
 
 const ns = "[Blueprint Table]";
-const deprec = `${ns} DEPRECATION:`;
-
-export const COLUMN_HEADER_CELL_MENU_DEPRECATED =
-    deprec + ` <ColumnHeaderCell> menu is deprecated. Use menuRenderer instead.`;
-export const COLUMN_HEADER_CELL_USE_INTERACTION_BAR_DEPRECATED =
-    deprec +
-    ` <ColumnHeaderCell> enableColumnInteractionBar is deprecated. Pass the prop to the top-level <Table> instead. When you pass the prop to <Table>, that value will override the one provided directly to <ColumnHeaderCell>.`;
-
-export const ROW_HEADER_CELL_MENU_DEPRECATED =
-    deprec + ` <RowHeaderCell> menu is deprecated. Use menuRenderer instead.`;
+// const deprec = `${ns} DEPRECATION:`;
```

Comment:
commented code

Reply:
eh, sort of on purpose. will be useful as a reminder for future deprecation messages 🤷‍♂️